src/main/java/de/embl/cba/mobie/image/MutableImageProperties.java

```diff
@@ -13,4 +13,5 @@
 	public ArrayList< Double > selectedLabelIds;
 	public boolean showSelectedSegmentsIn3d;
 	public boolean showImageIn3d;
+	public double[] addedTransform;
```

Comment:
We should add this to the metadata class too - https://github.com/tischi/imagej-utils/blob/master/src/main/java/de/embl/cba/bdv/utils/sources/Metadata.java

Reply:
This is done in imagej-utils 0.5.10-SNAPSHOT

Reply:
Ok! I don't see it on github though? Did you do it in a branch?

Reply:
Now?

Reply:
Yes! Can you add it to the copy() function too? https://github.com/tischi/imagej-utils/blob/master/src/main/java/de/embl/cba/bdv/utils/sources/Metadata.java#L90

Reply:
Done.